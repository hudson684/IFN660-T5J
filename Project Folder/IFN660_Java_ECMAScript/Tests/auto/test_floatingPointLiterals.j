1e1f
2.f
.3f
0f
3.14f
6.022137e+23f

0x0.1p0
0x11.11p0
0x1p3
0x1p5
0x.9p+812
0x.9p+81
0x2p+88F
0x1.p+88F
0x1111_32A9_B338.123p+88
0x1.9p-69f