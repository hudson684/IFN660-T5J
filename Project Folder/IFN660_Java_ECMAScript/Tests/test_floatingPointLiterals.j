1e1f
2.f
.3f
0f
3.14f
6.022137e+23f
0x1.9p-69f
0x.9p+812
0x2p+88F
0x1.p+88F
0x1111_32A9_B338.p+88F