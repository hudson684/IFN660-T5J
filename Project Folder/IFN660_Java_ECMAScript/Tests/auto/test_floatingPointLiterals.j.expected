FloatingPointLiteral (10)
FloatingPointLiteral (2)
FloatingPointLiteral (0.300000011920929)
FloatingPointLiteral (0)
FloatingPointLiteral (3.14000010490417)
FloatingPointLiteral (6.02213704067015E+23)
FloatingPointLiteral (0.0625)
FloatingPointLiteral (17.06640625)
FloatingPointLiteral (8)
FloatingPointLiteral (32)
FloatingPointLiteral (1.53631052533552E+244)
FloatingPointLiteral (1.36004154706646E+24)
FloatingPointLiteral (6.1897001964269E+26)
FloatingPointLiteral (3.09485009821345E+26)
FloatingPointLiteral (5.80766017035364E+39)
FloatingPointLiteral (2.64697796016969E-21)
EOF